### PR TITLE
groups: misc frontend fixes

### PR DESCRIPTION
**Pull request by liam-fitzgerald**
interface: fix unreads not clearing chats
 Fixes urbit/landscape#761

interface: inset by safe-area on iOS
    
Fixes urbit/landscape#242

interface: add disable spellcheck setting
    
Fixes urbit/landscape#79

interface: strip apps from landscape omnibox
 
Fixes urbit/landscape#1295

landscape: do not show cross-desk notifications

interface: prevent hook crash in publish notes
    
Fixes urbit/landscape#1299
Fixes urbit/landscape#1343

 interface: remove My Channels link from GroupSwitcher
    
 Fixes urbit/landscape#1331

### CODE REVIEW

File: pkg/interface/src/views/apps/chat/components/ChatWindow.tsx [hunk 1]
```diff
@@ -161,6 +161,14 @@ class ChatWindow extends Component<
     }
   }
 
+  onTopLoaded = () => {
+    const { graphSize, unreadCount } = this.props;
+    console.log(graphSize, unreadCount);
```
Comment (arthyn) [inline pkg/interface/src/views/apps/chat/components/ChatWindow.tsx:166]:
```suggestion
```

File: pkg/interface/src/views/apps/publish/components/Note.tsx [hunk 2]
```diff
@@ -148,4 +141,16 @@ export function Note(props: NoteProps & RouteComponentProps) {
   );
 }
 
-export default Note;
+export default function(props: NoteProps & RouteComponentProps) {
```
Comment (arthyn) [inline pkg/interface/src/views/apps/publish/components/Note.tsx:144]: Kinda curious why this fixes the hook crash

**Review by arthyn — APPROVED** [review]

File: pkg/interface/src/views/apps/publish/components/Note.tsx (hunk 2 above)
Comment (liam-fitzgerald) [reply pkg/interface/src/views/apps/publish/components/Note.tsx:144]: Rules of hooks, i.e. early returns

Comment (arthyn) [reply pkg/interface/src/views/apps/publish/components/Note.tsx:144]: oh derp yep lol

**Pull request merged**